doc/Install.md

```diff
@@ -79,24 +80,47 @@ In this case find the [mysqld] section in your my.cnf file and add the line :
 
 Restart mysql and you should be fine.
 
-
-### Run the installer
+### Option A: Run the manual installer
 
 Point your web browser to the new site and follow the instructions.
 Please note any error messages and correct these before continuing.
 
 If you need to specify a port for the connection to the database, you can do so in the host name setting for the database.
 
-*If* the automated installation fails for any reason, check the following:
+*If* the manual installation fails for any reason, check the following:
 
 * Does ".htconfig.php" exist? If not, edit htconfig.php and change the system settings. Rename to .htconfig.php
-* Is the database is populated? If not, import the contents of "database.sql" with phpmyadmin or mysql command line.
+* Is the database is populated? If not, import the contents of "database.sql" with phpmyadmin or the mysql command line.
 
 At this point visit your website again, and register your personal account.
 Registration errors should all be recoverable automatically.
 If you get any *critical* failure at this point, it generally indicates the database was not installed correctly.
 You might wish to move/rename .htconfig.php to another name and empty (called 'dropping') the database tables, so that you can start fresh.
 
+### Option B: Run the automatic install script
+
+Open the file htconfig.php in the main Friendica directory with a text editor. Remove the `die('...');` line and edit the lines to suit your installation (MySQL, language, theme etc.), then save the file (do not rename it). 
```

Comment:
Can you please stick the "One-Sentence-per-Line" rule for documentation. It makes the diffs much more readable, prevents long sentences and the output is rendered anyway so that the line-breaks do not show up.

Reply:
Fixed here https://github.com/friendica/friendica/pull/4916